src/NServiceBus.AcceptanceTests/Recoverability/When_cross_q_transactional_message_is_moved_to_error_queue.cs

```diff
@@ -10,12 +11,10 @@
 
     public class When_cross_q_transactional_message_is_moved_to_error_queue : NServiceBusAcceptanceTest
     {
-        [TestCase(TransportTransactionMode.SendsAtomicWithReceive)]
-        public async Task Should_not_dispatch_outgoing_messages(TransportTransactionMode transactionMode)
+        [Test]
+        public async Task Should_not_dispatch_outgoing_messages()
         {
-            Requires.DtcSupport();
-
-            var context = await Scenario.Define<Context>(c => { c.TransactionMode = transactionMode; })
+            var context = await Scenario.Define<Context>(c => { c.TransactionMode = TransportTransactionMode.SendsAtomicWithReceive; })
```

Comment:
No, the same test exists for `TransactionScope` at `When_transactional_message_is_moved_to_error_queue` but they were split from one test using `TestCase` into these to make them more accessible for downstreams.